### DISCUSSION TITLE
Logging Capture

### DISCUSSION THREAD

**Discussion by carldunham** [CleanCut/green#143] [Enhancements]
Is there a way to capture and display log output for errors? This is a very useful feature of other test runners/plugins.

**Comment by CleanCut** [comment]
Do you mean output sent through [Python's logging facility?](https://docs.python.org/2/library/logging.html)

**Comment by carldunham** [comment]
Exactly. nose and nose2 have something like this: http://nose2.readthedocs.io/en/latest/plugins/logcapture.html

**Comment by CleanCut** [comment]
The answer is no, we don't currently have that feature.  I wouldn't be opposed to it, if someone wanted to submit a patch with that kind of functionality.
